crates/tests/test-programs/src/prover.rs

```diff
@@ -7,27 +8,30 @@ fn main() -> Result<()> {
     gevulot_shim::run(run_task)
 }
 
-fn run_task(task: &Task) -> Result<TaskResult> {
+fn run_task(task: Task) -> Result<TaskResult> {
     println!("prover: task.args: {:?}", &task.args);
 
     //TODO define some provided context
-    let files = task.get_task_files_path("/workspace");
+    let files = task.get_task_files_path(WORKSPACE_PATH);
     println!("Prover: get nb files:{}", files.len());
-    for (name, path) in files {
-        let content = String::from_utf8(std::fs::read(path)?)?;
-        println!("Prover: Read file:{name} with content:{content:?}");
+    for (name, _path) in files {
+        // let content = String::from_utf8(std::fs::read(path)?)?;
+        // println!("Prover: Read file:{name} with content:{content:?}");
+        println!("Prover: Read file:{name}");
     }
-    let entries = fs::read_dir("/workspace")
+    let entries = fs::read_dir(WORKSPACE_PATH)
         .unwrap()
         .map(|res| res.map(|e| e.path()))
         .collect::<std::io::Result<Vec<_>>>()
         .unwrap();
     println!("file entries in /workspace :: {:?}", entries);
 
-    std::fs::write("/workspace/proof.dat", b"this is a proof.")?;
-
+    std::fs::write(format!("{WORKSPACE_PATH}/proof.dat"), b"this is a proof.")?;
     task.result(
         vec![0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
-        vec![String::from("/workspace/proof.dat")],
+        vec![
+            String::from(format!("{WORKSPACE_PATH}/proof.dat")),
+            String::from(format!("{WORKSPACE_PATH}/bigfile.bin")),
```

Comment:
Should this file be generated during runtime?

Reply:
I wanted an easy way to transfer a big file from prover to verifier for test, so I was thinking of using the same file was easier.
It was only for test, and at some point perhaps we should revert to a more playful example.

Reply:
Yeah, I get that. I was just thinking if this code works as-is on a new installation? I mean, would the end user need to put that "bigfile.bin" somewhere in order to make this work or does that somehow work already out-of-the-box?